docs/development/search.rst

```diff
@@ -0,0 +1,99 @@
+Search
+============
+
+Read The Docs uses Elasticsearch_ instead of built in Sphinx search for providing better search
+results. Documents are indexed in Elasticsearch index and the search is made through API.
+All the Search Code is Opensource and lives in `Github Repository`_.
+Currently we are using `Elasticsearch 6.3`_ version.
+
+Local Development Configuration
+-------------------------------
+
+Installing and running Elasticsearch
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+You must need to install and run `Elasticsearch 6.3`_ version in your local development machine.
```

Comment:
You need to install and run the Elasticsearch 6.3 version on your local development machine. You can get the installation instructions `here`.